packages/client/hmi-client/src/components/workflow/ops/compare-datasets/tera-compare-datasets-node.vue

```diff
@@ -1,30 +1,106 @@
 <template>
 	<section>
-		<tera-operator-placeholder :node="node">Attach datasets/simulation outputs to compare</tera-operator-placeholder>
-		<Button v-if="hasAtLeastTwoValues" label="Open" @click="emit('open-drilldown')" severity="secondary" outlined />
+		<NodeCharts
```

Comment:
Convention wise, we tend to use kebab-cases for our components. So `<node-charts...` instead of `<NodeCharts`. Also I know that @YohannParis liked to have a "tera-" prefix in front of our own components though I am not exactly sure what is the sentiment at this moment.